src/frontend/Komponenter/Behandling/TidligereVedtaksperioder/typer.ts

```diff
@@ -8,4 +10,25 @@ export interface ITidligereInnvilgetVedtak {
     harTidligereOvergangsstønad: boolean;
     harTidligereBarnetilsyn: boolean;
     harTidligereSkolepenger: boolean;
+    periodeHistorikkOvergangsstønad: IGrunnlagsdataPeriodeHistorikk[];
+}
+
+export interface IGrunnlagsdataPeriodeHistorikk {
+    antallMåneder: number;
+    antallMånederUtenBeløp: number;
+    vedtaksperiodeType: EPeriodetype;
+    fom: string;
+    tom: string;
+}
+
+export interface IStonader {
+    stønadstype: string;
+    viseInfotrygdKort?: boolean;
+    historikkISak?: IGrunnlagsdataPeriodeHistorikk[] | undefined;
+    verdier?: IVerdier;
```

Comment:
er feltet `verdier` nødvendig her? Kunne man bare kalt feltene for `harTidligereStønadISak` og `harTidligereStønadIInfotrygd` og lagt dem på samme nivå som de andre feltene?

Reply:
Generelt vil jeg også si at et felt i et objekt burde få navn basert på hvilken informasjon feltet tilbyr